llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -949,22 +964,32 @@ void RISCVFrameLowering::emitPrologue(MachineFunction &MF,
   }
 
   if (RVFI->useQCIInterrupt(MF)) {
+    // The function starts with `QC.C.MIENTER(.NEST)`, so the `(QC.)CM.PUSH(FP)`
+    // could only be the next instruction.
+    PossiblePush = std::next(PossiblePush);
```

Comment:
++PossiblePush?

Reply:
Happy to. I wasn't sure the idiomatic way to deal with iterators.